GigeVision.Core/Services/StreamReceiverParallelOpencv.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Emgu.CV;
```

Comment:
This class is supposed to be public

Reply:
And for the EMGU dependency there should be a new project that will only reference the Base library and provides the easy integration with EMGU/OpenCV Mat type